about/index.md

```diff
@@ -14,45 +14,31 @@ image:
 
 # Background and Objectives
 
-  Our goal is to transform plant breeding by advancing the science of phenotyping. Phenotypes are measurable features that indicate how they will grow and respond to stresses such as heat, drought, and pathogens.
+  The Transportation Energy Resources from Renewable Agriculture Phenotyping Reference Platform (TERRA-REF) will transform plant breeding by advancing the science of phenotyping. Phenotypes are measurable features that indicate how plants will grow and respond to stresses such as heat, drought, and pathogens.
   Breeding is currently limited by the speed at which phenotypes can be measured, and the information that can be extracted from these measurements.
   Currently, measurements used to predict yield include measuring leaf thickness with a caliper or height with a meter stick.
   More sophisticated instruments used to quantify plant architecture, carbon uptake, water use, and root growth do not scale to the thousands or tens of thousands of individual plants that need to be evaluated in a breeding program.
   
-We will develop an integrated phenotyping system for energy Sorghum that leverages genetics and breeding, automation, remote plant sensing, genomics, and computational analytics. The complete system will include: 1) the reference field phenotyping system (GFE gantry system) provided by ARPA-E; 2) field- and controlled-environment phenotyping of energy Sorghum; 3) computational solutions for phenotype selection and prediction; 4) genetics, genomics and bioinformatics for phenotype-to-genotype trait associations; 5) data processing and computational workflows for integrated phenotyping systems; and 6) phenotyping data and computational pipeline standards developed by a committee selected in collaboration with the TERRA program. 
+TERRA-REF will develop an integrated phenotyping system for energy Sorghum that leverages genetics and breeding, automation, remote plant sensing, genomics, and computational analytics. The complete system will include: 
+1) automated gantry field phenotyping system (LemnaTec field scanalyzer) provided by ARPA-E; 
```

Comment:
enumerated lists are formatted in Markdown like this:

> 1. first item 
2. second item

looks like

1. first item 
2. second item